src/Misc/PerformanceViewer/performanceViewerCollector.ts

```diff
@@ -69,7 +69,7 @@ export class PerformanceViewerCollector {
      * @param _scene the scene to collect on.
      * @param _enabledStrategyCallbacks the list of data to collect with callbacks for initialization purposes.
      */
-    constructor(private _scene: Scene, _enabledStrategyCallbacks?: PerfStrategyInitialization[]) {
+    constructor(private _scene: Scene, _enabledStrategyCallbacks?: {strategyCallback: PerfStrategyInitialization, category?: string}[]) {

```

Comment:
this can ca a type that you can reuse (you do need it further down the code, and as type for defaultStrategies)